Dockerfile

```diff
@@ -1,4 +1,10 @@
+FROM maven:3.6.3-jdk-8 AS serverBuild
+COPY frontend /frontend
```

Comment:
This only works when frontend is checked out inside this project first. 

Consider using GIT submodule, or having two docker images (frontend, backend) and orchestrating them through docker-compose.